app/measurement/views.py

```diff
@@ -255,7 +306,7 @@ class AggregatedViewSet(IsAuthenticated, viewsets.ViewSet):
         cannot be used
     '''
 
-    @swagger_auto_schema(
+    @ swagger_auto_schema(
```

Comment:
is this correct syntax with the space?

Reply:
Argh I don't know why my formatter keeps adding that